login.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PetPal - Log In</title>
+
+  <!-- Import Fonts and Icons -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.css">
+
+  <!-- Custom Styles -->
+  <link rel="stylesheet" href="css/base.css">
+  <link rel="stylesheet" href="css/global.css">
+  <link rel="stylesheet" href="css/login-signup.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <a href="/" class="logo">
```

Comment:
Change to # as well as to all other pages

Reply:
I was thinking since the logo actually has an index page now it makes sense for it to lead to the home page, same logic with other pages once they start leading somewhere we add them to the header.

Reply:
For the login/signup pages, logo should point to "/", but for the shelter and seeker povs, they should point to their own home page respectively. I will address this on my next iteration when i implement the home pages